src/main/resources/application.yaml

```diff
@@ -10,6 +10,10 @@ spring:
     url: jdbc:h2:file:./target/air-quality-hackathon-project
   h2:
     # enable H2 database web interface
-    console:
-      enabled: true
-      path: /system/db
+    #console:
+      #enabled: true
+      #path: /system/db
+      spring.h2.console.enabled=true
+      spring.h2.console.path=/h2-console
+
+
```

Comment:
this change is not needed AND wrong
in fact you are creating properties:

spring.h2.spring.h2.console.enabled=true
spring.h2.spring.h2.console.path=/h2-console

These are the wrong property names.
I guess in you old case it didn't work, because we forgot to disable security on /system/db endpoint.